### DISCUSSION TITLE
StepZen Stream on Friday at 12:00pm PT

### DISCUSSION THREAD

**Discussion by ajcwebdev** [stepzen-samples/stepzen-spacex-graphql#28] [Announcements]
Hello 👋! We'd like to highlight the work everyone has contributed to Hacktoberfest so far. On Friday at 12:00pm PT, Lucia and I will be on the StepZen Stream and any Hacktoberfest participants are welcome to join us to share their experience.

@Aayush0606 @gagandeepp @virtuoushub @rek990 @aloks98

If you would like to join us please respond to this discussion thread.

**Comment by rek990** [comment]
Hi @ajcwebdev I'll be there!

**Reply by ajcwebdev** [reply]
Awesome!

**Reply by rek990** [reply]
@ajcwebdev do we need to have our work ready to share on the screen, or is this more of a discussion format where we share our experiences? Thanks!

**Reply by ajcwebdev** [reply]
Sure, you are welcome to share whatever you are comfortable sharing. You can demonstrate your contribution and talk about the code or just talk about your experience. You could talk about what you took away from it, if there's anything that could be improved, your previous experience with open source, etc.

**Comment by ajcwebdev** [comment]
For @rek990 and anyone else who decides to join us tomorrow:

I will post a StreamYard link here about 5-10 minutes before the start of the stream tomorrow (stream starts at 12:00pm PT). You'll click the link to join the studio and then I will add you to the stage. If you want to join a little before the start of the stream we can do an audio and video check.

**Comment by virtuoushub** [comment]
👋🏻 @ajcwebdev - I plan to attend as well.

**Reply by ajcwebdev** [reply]
Awesome!

**Comment by ajcwebdev** [comment]
Here is the StreamYard link for anyone who would like to join us at the top of the hour:

https://streamyard.com/w67c84e2xz
